src/restful/controllers/authController.js

```diff
@@ -291,27 +302,23 @@ class AuthController {
               .collection("users")
               .doc(req.user.user_id);
             await docRef.set(
-              { first_name, last_name, bio, imageUrl },
+              { first_name, last_name, bio, imageUrl, showcase, credits, showreel },
```

Comment:
Had a chat with Imriel today and some thing will be changing.

In the doc, we want it to only have profile_meta. But profile_meta will expect different fields depending on the user.

For brands it will expect social media

For creators it will expect credits, showcase, social media and showreel